apps/meteor/client/views/admin/import/ImportHistoryPage.js

```diff
@@ -155,7 +155,7 @@ function ImportHistoryPage() {
 							</Table.Row>
 						)}
 					</Table.Head>
-					<Table.Body>
+					<Table.Body data-qa-id='ImportTable'>
```

Comment:
Its problably better set this data-qa-id in the `table` a and the later on use like `[data-qa-id="ImportHistoryPage"] tbody`, we should start use data-qa-id for "containers" elements